css/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>index</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <header>
+
+    <nav>
+
+      <img src="./assets/images/PlanIT_Logo.jpeg" alt="planIT_Logo" class="planIT-logo">
+
+      <ul class="navigation">
+        <li><a href="#Caracteristicas">Caracteristicas</a></li>
+        <li><a href="#Precio">Precio</a></li>
+        <li><a href="#Contacto">Contacto</a></li>
+      </ul>
```

Comment:
Check the font-weights and other font related properties for this items. There are some differences compared to the designs